packages/plugin-koa/src/koa.js

```diff
@@ -27,7 +27,11 @@ module.exports = {
         event.addMetadata('request', metadata)
       }, true)
 
-      await next()
```

Comment:
koa plugin wasn't using domain previously so with this change we now improve error capturing in koa significantly